src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -94,7 +88,7 @@ public BitbucketSCM(
                 new BitbucketSCMRepository(credentialsId, projectKey, repositorySlug, serverId));
         this.gitTool = gitTool;
         this.branches = branches;
-        this.extensions = extensions;
+        this.extensions = extensions == null ? Collections.singletonList(new BitbucketPostBuildStatus(serverId)) : extensions;
```

Comment:
IntelliJ warns me extensions is always null in this situation. Is that accurate?

Reply:
nope, and this is does not work either. It only appends our extension if no other extensions are in use.